modules/integration/src/test/java/org/apache/fluo/integration/client/FluoAdminImplIT.java

```diff
@@ -184,4 +193,90 @@ public void testInitializeLongChroot() throws Exception {
     }
   }
 
+
+  @Test
+  public void testRemove() throws Exception {
+
+
+    // oserver is started before every test so oserver is running
+    try (FluoAdmin admin = new FluoAdminImpl(config)) {
+      admin.remove();
+      fail("expected remove() to fail with oracle server running");
+    } catch (FluoException e) {
+    }
+
+    // TODO assert null or something below instead of sys.out
```

Comment:
Can these 2 new TODO statements be resolved before adding this code?

Reply:
Absolutely, I wast to use asserts and kill the sys.out statements I left from debugging.